picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -434,6 +434,21 @@ public Builder transform(Transformation transformation) {
       return this;
     }
 
+    /**
+     * Add a list of custom transformations to be applied to the image.
+     * <p>
+     * Custom transformations will always be run after the built-in transformations.
+     */
+    public Builder transform(List<? extends Transformation> transformations) {
```

Comment:
You want `List<? super Transformation>` here. Then you get `Transformation` objects without casting.